docs/user-docs/annotation.md

```diff
@@ -1337,6 +1345,7 @@ The following attributes can be used to manipulate the presentation settings of
       - `csv` for comma-seperated values.
       - `raw` for space-seperated values.
   - `"selector_ux_mode"`: The display mode for the recordedit input field when this column directive is a foreign key relationship. Supported values are `"facet-search-popup"` and `"simple-search-dropdown"`, with `"facet-search-popup"` being the default. Currently only supported in `entry` contexts.
+  - `"bulk_create_foreign_key"`: Use this property to control the bulk selection of foreign key values in `entry/create` context when there is a prefill query parameter. Supported values are a foreign key `name` in the format of `['schema_name', 'foreign_key_name']` from the schema document, `false`, and `null`. Using a foreign key name will use that foreign key as the one being bulk selected if that foreign key is in the visible columns list. `false` turns off the heuristics that trigger this feature. `null` will override inheritance for this property and use the default heuristics. This will override the `bulk_create_foreign_key` property defined in the display property of the foreign-key annotation. Currently only supported in `entry/create` context.
```

Comment:
Please also add this to `docs/user-docs/column-directive.md`, and don't forget to update the table of contents.

Reply:
This is not related to this line in the code, but I'm commenting this here since it's related to the annotation doc:

It seems like in the previous set of changes you used `compactSelectBulkForeignKey` instead of the proper context name in the "Context Names" section. Please also update that.